internal/biz/lookup.go

```diff
@@ -37,15 +37,36 @@ func (s *GetSubjectsUsecase) Get(ctx context.Context, req *v0.LookupSubjectsRequ
 	}
 
 	if req.Resource == nil {
+		s.log.WithContext(ctx).Infof("Missing Resource in request %v", req)
```

Comment:
this should be at error level the logging, info is usually for providing information. Same comment applies to all the logs, when ever we encounter an error, we log at error level.